code/mob/living/silicon/robot.dm

```diff
@@ -2308,6 +2309,45 @@
 			oil = 0
 			src.remove_stun_resist_mod("robot_oil", 25)
 
+	//#############################
+	//Begin Borg Death Alerts Procs
+	//#############################
+	proc/get_coords()
+		if (isrobot(src))
+			var/mob/living/silicon/robot/R = src
+			var/turf/T = get_turf(R)
+			if (istype(T))
+				return " at [T.x],[T.y],[T.z]"
```

Comment:
```suggestion
		var/turf/T = get_turf(src)
		if (istype(T))
			return " at [T.x],[T.y],[T.z]"
```
The type check seems pointless to me as it will always pass. And `src` is implicitly of the correct type here, no need to shove it into a different var.